### DISCUSSION TITLE
Deploying Outline with Authelia

### DISCUSSION THREAD

**Discussion by rllola** [outline/outline#2768] [Show and tell]
I have written an incomplete guide that show how to use Outline with Authelia as an OpenID provider. There is some resources that I have linked too.

I hope people will find it interesting. Someone in r/selfhost was suggesting using it as they have a beta support for it so I though why not give it a go.

https://medium.com/@Lola_Dam/guide-for-installing-outline-with-authelia-as-an-openid-provider-129a141f6090

**Comment by james-d-elliott** [comment]
I added this a while back, you're welcome to check over it @rllola, and outline are welcome to link directly to it:

https://www.authelia.com/integration/openid-connect/outline/
